src/ocm/components/clusterConfiguration/networkConfiguration/AvailableSubnetsControl.tsx

```diff
@@ -77,18 +97,13 @@ export const AvailableSubnetsControl = ({
           <Stack>
             {isDualStack ? (
               values.machineNetworks?.map((_machineNetwork, index) => {
+                const machineSubnets = index === 1 ? IPv6Subnets : IPv4Subnets;
                 return (
                   <StackItem key={index}>
                     <SelectField
                       name={`machineNetworks.${index}.cidr`}
-                      options={
-                        (index === 1 ? IPv6Subnets.length > 0 : IPv4Subnets.length > 0)
-                          ? [makeNoSubnetSelectedOption(hostSubnets)].concat(
-                              toFormSelectOptions(index === 1 ? IPv6Subnets : IPv4Subnets),
-                            )
-                          : [makeNoSubnetAvailableOption()]
-                      }
-                      isDisabled={!hostSubnets.length}
+                      options={buildOptions(machineSubnets)}
+                      isDisabled={isViewerMode || !hostSubnets.length}
```

Comment:
There is no scenario where the select would need to be disabled based on the `hostSubnets`, so the `isDisabled` props 
 got removed from the selects in https://github.com/openshift-assisted/assisted-ui-lib/pull/1438. You can remove it from the conditions here, too.

Reply:
> Probably the only condition should be `isDisabled={isViewerMode}`

That's exactly what I meant :slightly_smiling_face:

Reply:
I agree, once the code is updated in `master` I'll adjust this PR.